metabrainz/templates/payments/payment.html

```diff
@@ -90,9 +90,10 @@ <h1 class="page-title">Make a Payment</h1>
 
     <h2>Other ways to pay</h2>
 
-    <h3>Bitcoin</h3>
-    <p>If you would like to make a payment in bitcoins, please use this address:</p>
-    <p><em style="margin-left: 2em">1Pc7sFnuRfDQ2rGoNwRA3nBX1Pqjjqb15v</em></p>
+    <h3>Bank transfer</h3>
+    <p>You can make a payment, or recurring payments, via bank transfer to the following IBAN:</p>
+    <p><em style="margin-left: 2em">ES65 2013 0614 9902 0060 1916</em></p>
+    <p>Please indicate your name and use the word <em>PAYMENT</em> in the description field.</p>
```

Comment:
Change this line to:

"Please specify the invoice number(s) you are paying in the description field of the transfer."